src/components/history-query-filters.vue

```diff
@@ -17,7 +17,13 @@
     props: {
       filtersList: {
         type: Array as PropType<HistoryQuery['selectedFilters']>,
-        required: true
+        required: true,
+        default: () => []
```

Comment:
With this we don't need to check if `filtersList` exists in line 2, remove it and leave it as
```
        <div v-if="filtersList.length"
```